### ISSUE TITLE
MCMC posterior distributions problem

### ISSUE THREAD

**Issue by gokhannY** [phoebe-project/phoebe2#954]

**Comment by amiszuda** [comment]
In general, whether using PHOEBE or another code, the key to understanding the dataset-dependent values of  q  and  a\sin i  lies in the data itself. While these parameters primarily manifest in radial velocities (RVs), they do influence light curves—albeit to a lesser extent. Given the significantly larger quantity of photometric data compared to RVs, as well as the respective weighting of each, it’s natural for the photometric solution to have a stronger influence on these values. I’m not certain if PHOEBE automatically handles such cases by assigning different weights to balance the impact of each dataset on the respective parameters (@kecnry @aprsa), but in my experience, fixing  q  and  a\sin i  to their RV-determined values and adding small priors during the MCMC process does the trick.

**Comment by gokhannY** [comment]
I agree with you. That's why my Gaussian around 0.1 and 1 for q and asini, respectively. I might try 0.02 for q, but the MCM calculation takes almost five days for this system.  

I also attached the trace graph. The starting points are all over the place.
[trace.pdf](https://github.com/user-attachments/files/17278048/trace.pdf)

**Comment by kecnry** [comment]
I agree with @amiszuda - you have a few options within phoebe to address this:
* find what other parameter(s) _might_ be able to account for the LC residuals that are "incorrectly" driving `asini` and `q`.  This may or may not be possible depending on the situation.
* account for potential underestimation of the uncertainties on the photometry points, using `sigmas_lnf` (I guess we don't have a great tutorial on this still, but it is explained in the [2.3 release paper](https://phoebe-project.org/publications/2020Conroy+)).  I'm not sure it will completely avoid the problem you're seeing, though.
* do an MCMC run on the RVs alone and then hold `q` and `asini` fixed for the MCMC run on LCs+RVs.  You can flip the constraint so that the second run can still vary the inclination while conserving `asini`.
* better yet (in my opinion), do an MCMC run on the RVs alone, and use those _posteriors_ as the _priors_ on the following MCMC run on LCs alone.  This effectively treats the information from all the RVs equally as the information from all the LCs.
* use a [custom cost function](https://phoebe-project.org/docs/latest/tutorials/emcee_custom_lnprob) to manually override the weights of each point (I'm not a fan of this one, but it is technically possible).

Whatever you end up doing, since this is somewhat "non-standard", I think its important to be very clear in any publication of exactly what you did and why so that the results are reproducible and taken in the correct context.
